android/src/main/java/com/reactnativemembrane/Utils.kt

```diff
@@ -46,15 +50,56 @@ internal fun listToRNArray(array: List<Any>): WritableArray {
   return res
 }
 
-internal fun mapToRNMap(map: Map<String, Any>): WritableMap {
+internal fun rtcOutboundStatsToRNMap(stats: RTCOutboundStats): ReadableMap{
+  val innerMap = Arguments.createMap()
+  innerMap.putDouble("bandwidth", if (stats.qualityLimitationDurations?.bandwidth != null) stats.qualityLimitationDurations!!.bandwidth else  0.0)
+  innerMap.putDouble("cpu", if (stats.qualityLimitationDurations?.cpu != null) stats.qualityLimitationDurations!!.cpu else  0.0)
+  innerMap.putDouble("none", if (stats.qualityLimitationDurations?.none != null) stats.qualityLimitationDurations!!.none else  0.0)
+  innerMap.putDouble("other", if (stats.qualityLimitationDurations?.other != null) stats.qualityLimitationDurations!!.other else  0.0)
+
+  val res = Arguments.createMap()
+  res.putString("kind", stats.kind)
+  res.putString("rid", stats.rid)
+  res.putInt("bytesSent",  if (stats.bytesSent?.toInt() != null) stats.bytesSent!!.toInt() else  0)
+  res.putDouble("targetBitrate", if (stats.targetBitrate != null) stats.targetBitrate!! else  0.0)
```

Comment:
I see that the targetBitrate always equals 0 for outbound tracks on Android. Is that correct or a bug?

Reply:
From what I could tell, even in native apps it is always 0 for video streams and non zero for audio